src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt

```diff
@@ -51,6 +51,21 @@ interface SessionTemplateRepository : JpaRepository<SessionTemplate, Long> {
     @Param("visitsToDate") visitsToDate: LocalDate?,
   ): Int
 
+  @Query(
+    "select count(*) from visit v " +
+      " JOIN session_template st ON st.reference = v.session_template_reference " +
+      " WHERE st.reference = :reference" +
+      " AND v.visit_start > :visitsFromDate" +
```

Comment:
should it be >= :visitsFromDate?

Reply:
This is tried and tested code... so we are already using this logic... will double check